Monika After Story/game/script-holidays.rpy

```diff
@@ -3042,9 +3809,11 @@ init 5 python:
 
 label monika_nye_year_review:
     $ persistent._mas_nye_spent_nye = True
+    $ spent_an_event = False
 
-    # starting with an overview based on time
-    if store.mas_anni.anniCount() >= 0:
+    $ _and = "and"
```

Comment:
underscore-prefixed short-named variables in a global scope caused a crash that took hours to figure out one time. This happens if the underscore-prefixed variable shares its name with a renpy variable. (think `_return`). Best to avoid using underscore-prefixed variables outside of functions.